### PR TITLE
Pull request for improvement/models in main application

### PR DESCRIPTION
**Pull request by AndriiKatsubo**

### CODE REVIEW

File: MedicalDocManagment.WebUI/app/admin/models/usersmodel.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+﻿import UserModel from '../models/usermodel';
 
 export default class UsersModel extends Array<UserModel> {
-    constructor(jsonobject?) {
+    constructor(jsonObject?) {
         super();
-        if (jsonobject) {
-            jsonobject.map((user) => { this.push(new UserModel(user)); });
+        if (jsonObject) {
+            jsonObject.forEach((user) => { this.push(new UserModel(user)); });
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/admin/models/usersmodel.ts:7]: I had trouble with same piece of code before. Does it work on your side?

**Review by KostetskyiYaroslav — APPROVED** [review]

File: MedicalDocManagment.WebUI/app/admin/models/usersmodel.ts (hunk 1 above)
Comment (AndriiKatsubo) [reply MedicalDocManagment.WebUI/app/admin/models/usersmodel.ts:7]: Yes, it works on my side — I checked features "users list" and "search user" and they are working.
Where exactly did you have troubles?

**Review by Ravennod — APPROVED** [review]

**Review by KostetskyiYaroslav — CHANGES_REQUESTED** [review]
This is bad code.
It doesn't  work

**Comment by KostetskyiYaroslav** [comment]
Не мерджи це поки що

**Comment by AndriiKatsubo** [comment]
> This is bad code.
It doesn't work

When it doesn't works?

**Review by KostetskyiYaroslav — APPROVED** [review]

**Pull request merged**